### PR TITLE
Created Tournament service interface + implementation

### LINKED ISSUE #64: [TASK] Create a Service interface + implementation for Tournament

### PR DESCRIPTION
**Pull request by Fuyunori**
Closes #64 

Currently still a work-in-progress, checklist of things to-do:
- [x] interface
- [x] basic implementation

### CODE REVIEW

File: service/src/main/java/tennisclub/service/TournamentService.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package tennisclub.service;
+
+import tennisclub.entity.Tournament;
+import tennisclub.entity.ranking.Ranking;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
```
Comment (mazanec202) [inline service/src/main/java/tennisclub/service/TournamentService.java:8]: In the reference eshop project they used the @Service annotation also for the interface, I guess we should add it here as well.

File: service/src/main/java/tennisclub/service/TournamentService.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    void create(Tournament tournament);
+    Tournament update(Tournament tournament);
+    void remove(Tournament tournament);
+    List<Tournament> listAll();
+    List<Tournament> listByStartTime(LocalDateTime startTime);
+    List<Tournament> listByEndTime(LocalDateTime endTime);
+    List<Tournament> listByTimeInterval(LocalDateTime from, LocalDateTime to);
+    List<Tournament> listByCapacity(Integer capacity);
+    void addRanking(Tournament tournament, Ranking ranking);
+}
```
Comment (mazanec202) [inline service/src/main/java/tennisclub/service/TournamentService.java:19]: Same as for the Lesson service, I'm missing here a method findById() which you probably planned to add later :)

**Review by mazanec202 — APPROVED** [review]

File: service/src/main/java/tennisclub/service/TournamentService.java (hunk 1 above)
Comment (Fuyunori) [reply service/src/main/java/tennisclub/service/TournamentService.java:8]: Thanks for pointing this out! 👍

File: service/src/main/java/tennisclub/service/TournamentService.java (hunk 2 above)
Comment (Fuyunori) [reply service/src/main/java/tennisclub/service/TournamentService.java:19]: Thanks! 😄

**Pull request merged**